arnab_bhadra/arnabbhadra_PostgresAssignment/src/server.ts

```diff
@@ -0,0 +1,4 @@
+import {app} from "./Application/app";
+app.listen(3000,()=>{
```

Comment:
move port to config file . ref https://github.com/adityajawade93/freshers-june2021/tree/main/jatin/postgres_assignment